cylc/flow/task_pool.py

```diff
@@ -1495,20 +1517,6 @@ def force_trigger_tasks(
             )
             # Flow values already validated by the trigger client.
             if itask is None:
-                if flow[0] == FLOW_ALL:
-                    flow_nums = self._get_active_flow_nums()
-                elif flow[0] == FLOW_NEW:
-                    flow_nums = {self.flow_mgr.get_new_flow(flow_descr)}
```

Comment:
Because this was within the `for name, point in future_tasks` loop this was creating a new flow for each triggered task rather than for each trigger event.